be/src/olap/rowset/rowset_meta_manager.cpp

```diff
@@ -38,11 +38,7 @@ bool RowsetMetaManager::check_rowset_meta(OlapMeta* meta, TabletUid tablet_uid,
                                           const RowsetId& rowset_id) {
     std::string key = ROWSET_PREFIX + tablet_uid.to_string() + "_" + rowset_id.to_string();
     std::string value;
```

Comment:
No need to get value?

Reply:
For doris, no need to get a value kere.
But rocksdb's ```KeyMayExist``` need a value.

Reply:
Yes,you are right